src/test/unit/helperFunctions/test_task_conversion.py

```diff
@@ -62,3 +65,4 @@ def test_convert_analysis_task_to_firmware_object(self):
         self.assertEqual(len(fw_obj.scheduled_analysis), 2)
         self.assertIn('dummy', fw_obj.scheduled_analysis)
         self.assertIsInstance(fw_obj.tags, dict, 'tag type not correct')
+        assert list(fw_obj.tags.keys()) == ['a', 'b'], 'tags not correct'
```

Comment:
Where does `'a'` and `'b'` come from?

Reply:
They are assigned by the `convert_analysis_task_to_fw_obj` call.